examples/resources/nobl9_slo/resource.tf

```
@@ -123,3 +123,46 @@ resource "nobl9_slo" "this" {
     }
   }
 }
+
+resource "nobl9_slo" "composite_slo" {
+  name             = "${nobl9_project.this.name}-composite"
+  service          = nobl9_service.this.name
+  budgeting_method = "Occurrences"
+  project          = nobl9_project.this.name
+
+  depends_on = [nobl9_slo.slo_1, nobl9_slo.slo_2]
+
+  time_window {
+    unit       = "Day"
+    count      = 3
+    is_rolling = true
+  }
+
+  objective {
+    display_name = "OK"
+    name         = "tf-objective-1"
+    target       = 0.8
+    value        = 1
+    composite {
+      max_delay = "45m"
+      components {
+        objectives {
+          composite_objective {
+            project      = nobl9_slo.slo_1.project.name
```

Right now error appears on apply:
```
 Error: Unsupported attribute
│ 
│   on main.tf line 170, in resource "nobl9_slo" "composite_slo":
│  170:             project      = nobl9_slo.slo_1.project.name
│ 
│ Can't access attributes on a primitive-typed value (string).
```
```suggestion
            project      = nobl9_slo.slo_1.project
```

---

Good catch, I modified it after applying, thanks! :smile: